Yoast/Yoast-SEO-for-TYPO3
# [BUGFIX] Fixed loading state of internal linking suggestions when bodytext is empty (e.g. new content element)

## Summary

This PR can be summarized in the following changelog entry:

* When the bodytext was empty, the internal linking suggestions would be kept in a loading state because the loading state is triggered with an empty "content" variable. This PR fixes that.

## Relevant technical choices:

* Instead of checking if the `content` is empty, the `getCKEditorContent` now returns `null` if the CKEditor has not yet finished loading. This way, it's still possible to have empty content but the loading state is not dependent on it anymore. 

## Test instructions

This PR can be tested by following these steps:

* Pull branch and check if the loading state now disappears when CKeditor is done loading

## Quality assurance


Fixes #536
